src/functions/postResult/application/save-result-service.ts

```diff
@@ -7,12 +7,15 @@ import { buildTestResultInsert, buildUploadQueueInsert } from '../framework/data
 export const saveTestResult = async (
   testResult: StandardCarTestCATBSchema,
   hasValidationError: boolean = false,
+  isPartialTestResult: boolean,
 ): Promise<void> => {
   const connection: mysql.Connection = getConnection();
-
   try {
-    await connection.promise().query(buildTestResultInsert(testResult, hasValidationError));
-    await trySaveUploadQueueRecords(connection, testResult, hasValidationError);
+    connection.query(`SET SESSION TRANSACTION ISOLATION LEVEL SERIALIZABLE;`);
```

Comment:
should this be: ```await connection.promise().query(...)``` ? 
we need to make sure that his command will be executed sequentially before the next one ?!

Reply:
changed to `await connection.promise().query()`